core/src/main/java/io/questdb/cairo/ColumnPurgeOperator.java

```diff
@@ -141,7 +145,7 @@ public boolean purge(ColumnPurgeTask task, TableReader tableReader) {
     public void purgeExclusive(ColumnPurgeTask task) {
         assert task.getTableName() != null;
```

Comment:
nit:
```java
assert scoreboardUseMode == ScoreboardUseMode.STARTUP_ONLY;
```

Reply:
rather `ScoreboardUseMode.VACUUM_TABLE`

Reply:
removed this method here: https://github.com/questdb/questdb/pull/5664/commits/f9090df3b37e702881d7356caeab5286c69fc623